### DISCUSSION TITLE
Slow Response using theme

### DISCUSSION THREAD

**Discussion by Benjamin-Medina** [Akifyss/obsidian-border#26] [General]
Hello, I love your theme. 

Unfortunately, when it's enabled in my Obsidian vault, the response time for everything slows dramatically. I restarted the app and still the lag occurs. Then I enabled my normal theme, Minimal, and the speed of the app returned to normal.

Is this to be expected? Am I alone in this experience of application responsiveness?  

All the best!

**Comment by Akifyss** [comment]
Hi, it is true that under normal circumstances the interface of the applied border theme will be less responsive than the default theme. However, I feel that its performance is acceptable to me and is unlikely to affect normal usage. Here is how the border theme works on my computer, I hope it can give you a reference：

![1](https://user-images.githubusercontent.com/23567154/217043548-5857b715-6b4e-49fd-8cf9-aa416bcd211d.gif)

If your situation is very unusual from the above picture, please provide me with more information.

**Reply by Benjamin-Medina** [reply]
Thanks mate, for the quick reply, and video. 

Here's a video of my experience. It's a bit quicker after I refreshed my computer, but this speed is with Obsidian as the only app open on my computer 

https://user-images.githubusercontent.com/118020281/217328043-ab785a23-f976-4bb6-bc49-a31823935f42.mov

Perhaps the lag is my operating system...

**Reply by Akifyss** [reply]
That's weird, I'm not sure if it's OS related or not. But you could try using the following theme file to see if it helps, this file excludes code snippets related to macos.

[theme.zip](https://github.com/Akifyss/obsidian-border/files/10684538/theme.zip)

**Comment by XIAOLEI-O** [comment]
Hello, I like your theme, almost perfect experience. I have tried a lot of functions, but there is a problem that cannot be solved. How to change the background color of the hover preview?

**Reply by Akifyss** [reply]
Hi, do you mean the background color of this part？
![image](https://user-images.githubusercontent.com/23567154/219379965-b88e3ed4-c7b0-4fc2-8e91-2464edde7cf4.png)

**Comment by XIAOLEI-O** [comment]
Yes, the gray background will have an impact, if it is a white background, the information will be displayed more clearly.


志尚
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Akifyss/obsidian-border"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月16日(星期四) 晚上9:57
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Akifyss/obsidian-border] Slow Response using theme (Discussion #26)





 
Hi, do you mean the background color of this part？

**Reply by Akifyss** [reply]
Currently the theme cannot modify the background color of the hover preview. But you can import this CSS snippet yourself, and then modify it in style settings.
[hover-preview-background.zip](https://github.com/Akifyss/obsidian-border/files/10757999/hover-preview-background.zip)

**Reply by XIAOLEI-O** [reply]
Thank you. It's very helpful to me.
